src/lib/MapCards.svelte

```diff
@@ -89,6 +91,7 @@
                 map.versions[0].hash
               }.jpg`}
               alt={map.name}
+              class:blur={isNsfw(map)}
```

Comment:
With the suggestion above doing the following might be slimmer code-wise:

```suggestion
              class:blur={map.nsfw}
```

or if you wanted to be as strict about requiring a true (which might be better in that case you can do as in your function above:

```suggestion
              class:blur={map.nsfw === true}
```

But not a necessary change. Go with how you prefer it.

Good use of built-in svelte features and smart implementation though!